### PR TITLE
Add escalation policy and clarify priority definitions

### PR DESCRIPTION
**Pull request by RNHTTR**

### CODE REVIEW

**Review by collinmcnulty — APPROVED** [review]

File: astro/astro-support.md [hunk 1]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+Here are the four ticket priorities with descriptions and examples for each:
+
+### **P1:** Critical impact
+
+A Deployment is completely unavailable, or a DAG that was previously working in production is no longer working.
+
+P1 tickets are handled with the highest levels of urgency, if Astronomer Support responds on a P1 ticket and subsequently does not hear back for 2 hours, the ticket priority will be automatically changed to P2.
+
+Additionally, if the immediate problem is solved but there are follow up investigations ongoing, those follow ups will be conducted in a separate ticket at a lower priority.
```
Comment (lzdanski) [inline astro/astro-support.md:93]:
```suggestion
Additionally, if the immediate problem is solved, but follow-up investigations continue, those investigations are conducted in a separate ticket at a lower priority.
```

File: astro/astro-support.md [hunk 2]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+### P2: High impact
+
+Ability to use Astro is severely impaired but does not affect critical, previously working pipelines in production.
```
Comment (lzdanski) [inline astro/astro-support.md:97]:
```suggestion
Your ability to use Astro is severely impaired, but does not affect any critical, previously working pipelines in production.
```

File: astro/astro-support.md [hunk 3]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+Examples:
+
+- A newly deployed production DAG is not working, even though it ran successfully in a development or test environment.
+- The Airflow UI is unavailable.
+- You are unable to deploy code to your Deployment, but existing DAGs and tasks are running as expected.
```
Comment (lzdanski) [inline astro/astro-support.md:103]:
```suggestion
- You can't deploy code to your Deployment, but existing DAGs and tasks run as expected.
```

File: astro/astro-support.md [hunk 4]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+Additionally, if the immediate problem is solved but there are follow up investigations ongoing, those follow ups will be conducted in a separate ticket at a lower priority.
+
+### P2: High impact
+
+Ability to use Astro is severely impaired but does not affect critical, previously working pipelines in production.
+
+Examples:
+
+- A newly deployed production DAG is not working, even though it ran successfully in a development or test environment.
```
Comment (lzdanski) [inline astro/astro-support.md:101]:
```suggestion
- A newly deployed production DAG is not working, even though it worked successfully in a development or test environment.
```

File: astro/astro-support.md [hunk 5]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that is required for running tasks, such as adding a new worker instance type.
+- Task logs are missing in the Airflow UI.
+
+### P3: Medium impact
+
+Service is partially impaired.
+
+Examples:
+
+- A newly deployed DAG is not working in a development Deployment, even though it ran successfully in a local environment using the Astro CLI.
```
Comment (lzdanski) [inline astro/astro-support.md:113]:
```suggestion
- A newly deployed DAG is not working in a development Deployment, even though it worked successfully in a local environment using the Astro CLI.
```

File: astro/astro-support.md [hunk 6]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that affects your cluster's performance but isn't required to run tasks, such as changing the size of your cluster's database or adding a new VPC peering connection.
```
Comment (lzdanski) [inline astro/astro-support.md:114]:
```suggestion
- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that affects your cluster's performance, but isn't required to run tasks, such as changing the size of your cluster's database or adding a new VPC peering connection.
```

File: astro/astro-support.md [hunk 7]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+- Astro CLI usage is impaired (for example, there are incompatibility errors between installed packages).
```
Comment (lzdanski) [inline astro/astro-support.md:115]:
```suggestion
- Astro CLI usage is impaired. For example, there are incompatibility errors between installed packages.
```

File: astro/astro-support.md [hunk 8]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+- There is an Airflow issue that has a code-based solution.
+- You received a log alert on Astronomer.
+- You have lost the ability to use a [Public Preview](https://docs.astronomer.io/astro/feature-previews) feature that does not affect general services.
```
Comment (lzdanski) [inline astro/astro-support.md:118]:
```suggestion
- You lost the ability to use a [Public Preview](https://docs.astronomer.io/astro/feature-previews) feature that does not affect general services.
```

File: astro/astro-support.md [hunk 9]
```diff
@@ -114,6 +78,70 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+### P4: Low impact
+
+Astro is fully usable but you have a question for our team.
```
Comment (lzdanski) [inline astro/astro-support.md:122]:
```suggestion
Astro is fully usable, but you have a question for the Support team.
```
Does this need to be specific? Support? Dev? General team?

**Review by lzdanski — CHANGES_REQUESTED** [review]
These are mostly style and voice changes! The commit I made makes the severity examples actual headers (H3 level) instead of just **bold**, so that you can link directly to them from a ticket if you need to explain why something was changed from P2 to P3 or a similar scenario.

File: astro/astro-support.md [hunk 10]
```diff
@@ -114,6 +78,62 @@ To add a teammate to an existing support request, cc them when replying on the s
 
 :::
 
+## Ticket Priorities
+
+To help Astronomer Support respond effectively to your support request, priorities are determined automatically by Astronomer. You can read more about ticket priorities and their SLAs [here](https://www.astronomer.io/legal/technical-support-success-packages/).
+
+Here are the four ticket priorities with descriptions and examples for each:
```
[the remark this answers is not in the document]
Comment (RNHTTR) [reply astro/astro-support.md:85]: I don't think this is accurate. We have 4 priorities (P1 through P4), and there are examples for each, but the priorities themselves aren't examples.

File: astro/astro-support.md [hunk 11]
```diff
@@ -114,6 +78,68 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+
+Examples:
+
+- There are package incompatibilities caused by a specific, complex use case.
+- You have an inquiry or a small bug report for a Public Preview feature.
+
+## Request an escalation for an existing support ticket
+<BusinessBadge/>
+
+Business and Enterprise customers can request escalated support for an existing ticket. To request a support escalation, email escalations@astronomer.io with the following information:
```
Comment (pgagnon) [inline astro/astro-support.md:132]: I think we clarify what outcome customers can expect from an escalation.

File: astro/astro-support.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (lzdanski) [reply astro/astro-support.md:85]: Here's a take-two on it!

File: astro/astro-support.md (hunk 11 above)
Comment (RNHTTR) [reply astro/astro-support.md:132]: how's this?

File: astro/astro-support.md [hunk 12]
```diff
@@ -114,6 +78,71 @@ To add a teammate to an existing support request, cc them when replying on the s
...
+* Issue summary
+* Reason for escalation
+
+Reasons for escalation include:
+
+* The ticket is taking much longer than expected to resolve.
+* Multiple attempts to solve the issue have failed.
+* The urgency of the ticket has increased substantially.
+
+When an escalation request is made, the on-call shift manager will be alerted to the ticket and will evaluate the escalation request. The on-call shift manager will either assume ownership of the ticket or ensure meaningful and timely progress is made on the ticket.
```
Comment (lzdanski) [inline astro/astro-support.md:144]:
```suggestion
When you make an escalation request, it alerts the on-call shift manager to the ticket, who evaluates the escalation request. The on-call shift manager either assumes ownership of the ticket or ensures meaningful and timely progress is made on the ticket.
```

File: astro/astro-support.md (hunk 11 above)
Comment (pgagnon) [reply astro/astro-support.md:132]: 👍

**Review by lzdanski — DISMISSED** [review]
One final suggestion!

**Review by lzdanski — APPROVED** [review]

**Pull request merged**